packages/grafana-ui/src/components/Table/utils.ts

```diff
@@ -130,7 +130,7 @@ export function getColumns(
       Cell,
       id: fieldIndex.toString(),
       field: field,
-      Header: getFieldDisplayName(field, data),
+      Header: fieldTableOptions.hideHeader ? () => null : getFieldDisplayName(field, data),
```

Comment:
nitpick: wonder if we need a `() => null` function here? does empty string or null value not work as well?